prerendering.bs

```diff
@@ -658,6 +673,33 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+[=Accept-CH cache=], which is owned by user agent as a global storage, should not be modified by [=prerendering browsing context=].
+
+Instead, each [=prerendering browsing context=] maintains its <dfn>prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields. 
+
+Similar to <a spec=CLIENT-HINTS-INFRASTRUCTURE> add a new Accept-CH cache entry</a>,
+to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerender scoped Accept-CH cache=],
+given an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] [=prerender scoped Accept-CH cache=][|origin|] to |hintSet|.
+
+<div algorithm="append client hints to request patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>append client hints to request</a> algorithm, by prepending the following steps before iterating over the |hintSet|:
+
+  1. If [=browsing context=] is a [=prerendering browsing context=], retrieve |prerender client hint set| in [=prerender scoped Accept-CH cache=], and then update the |hintSet| with |prerender client hint set|.
+  
+</div>
+
+<div algorithm="update the Client Hints set patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>update the Client Hints set</a> algorithm, by updating the last step.
+
+  1. If [=browsing context=] is a [=prerendering browsing context=],
```

Comment:
This should be the `|browsingContext|` var, not the `[=browsing context=]` dfn, right?

Reply:
indeed . 👍